package.json

```diff
@@ -113,7 +113,7 @@
     "eslint-config-react-app": "^5.1.0",
     "eslint-loader": "^3.0.3",
     "eslint-plugin-chai-friendly": "^0.5.0",
-    "eslint-plugin-compat": "^3.3.0",
+    "eslint-plugin-compat": "~3.3.0",
```

Comment:
Pinned in this PR becase the latest version of eslint-plugin-compat shows error.